pkg/vm/context/cairo_run.go

```diff
@@ -0,0 +1,49 @@
+package context
+
+import (
+	"github.com/lambdaclass/cairo-vm.go/pkg/parser"
+	"github.com/lambdaclass/cairo-vm.go/pkg/runners"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm"
+)
+
+type RunResources struct {
+	NSteps *uint
+}
+
+type BuiltinHintProcessor struct {
+	//Todo: ExtraHints update to map[string][dyn func]
+	ExtraHints   map[string]string
+	RunResources *RunResources
+}
+
+type CairoRunConfig struct {
+	Entrypoint   string
+	TraceEnabled bool
+	RelocateMem  bool
+	Layout       string
+	ProofMode    bool
+	SecureRun    *bool
+}
+
+func CairoRunProgram(compiledProgram parser.CompiledJson) error {
+	hintExecutor := BuiltinHintProcessor{}
+	cairoRunConfig := CairoRunConfig{}
+	_, _, err := CairoRun(compiledProgram, hintExecutor, cairoRunConfig)
+	return err
+}
+
+func CairoRun(compiledProgram parser.CompiledJson, hintExecutor BuiltinHintProcessor, cairoRunConfig CairoRunConfig) (runners.CairoRunner, vm.VirtualMachine, error) {
+	programJson := vm.DeserializeProgramJson(compiledProgram)
+	vm := vm.NewVirtualMachine()
```

Comment:
`CairoRunner` already has a `VirtualMachine`, we should remove this line

Reply:
Done